esti/catalog_export_test.go

```
@@ -286,15 +286,16 @@ func TestAWSCatalogExport(t *testing.T) {
 
 	t.Run("symlink_exporter", func(t *testing.T) {
 		var columns = []string{"name", "color"}
+		csvData := genCSVData(columns, 3)
```

can you add error check to genCSVData - write can fail and we ignore the error